src/util/createStoreServer.js

```diff
@@ -30,7 +30,7 @@ export const serverFetchQueries = request => () => queries =>
  */
 export function getServerCreateStore(routes, middleware, request, baseUrl) {
 	const middlewareToApply = [
-		catchMiddleware,
+		catchMiddleware(err => request.server.app.logger.error(err)),
```

Comment:
Are these changes related, or just making sure we are actually console logging errors? :)